Header header
int64 id
float64 thetaMin
float64 thetaMax
float64 obstacleProjection
float64 centroidX
float64 centroidY
float64 centroidZ
